### DISCUSSION TITLE
How to run materialize/cli

### DISCUSSION THREAD

**Discussion by DuyTC1811** [MaterializeInc/materialize#14938] [Q&A] [answered]
Hello everyone

I am facing a problem when I am trying to run materialize/cli
docker-compose run cli  and docker run docker-cli-1   I ran this command line 
<img width="1440" alt="Screen Shot 2022-09-22 at 23 39 20" src="https://user-images.githubusercontent.com/110332852/191804263-b4ea3b07-1459-46e3-bd5e-9e927884435f.png">

and i realized that my materialize/cli:v0.26.4 is not working
how can i run it

**Comment by morsapaes** [comment]
Hi, @DuyTC1811! I'd expect it to work with:

`docker-compose run docker-cli-1`

As a sanity check, I just spin up a minimal `docker-compose` setup using:

```bash
services:
  materialized:
    image: materialize/materialized:v0.26.4
    ports:
      - "6875:6875"
  cli:
    image: materialize/cli:v0.26.4
```

, which is working as expected using:

`docker-compose up -d`

`docker-compose run cli`

Could you give it a new try and let us know whether it worked for you?

**Reply by DuyTC1811** [reply]
Thank you for responding
<img width="486" alt="Screen Shot 2022-09-23 at 07 29 05" src="https://user-images.githubusercontent.com/110332852/191874221-91d81a69-24e6-45ca-92c9-532e72bfc8fe.png">
I tried re-running `docker-comp run cli` it still doesn't work,   notify no configuration file provided: not found
<img width="613" alt="Screen Shot 2022-09-23 at 07 40 44" src="https://user-images.githubusercontent.com/110332852/191874341-3b706dcb-e13a-464f-b878-7914a92ecbe2.png">

**Comment by bobbyiliev** [comment, marked as answer]
Hi there @DuyTC1811,

As you are using a non-standard docker-compose yaml file name, you would also need to pass it as an argument to the `docker-compose run `command, eg:

```
docker-compose -f docker-materialize.yaml run cli
```

Let me know how it goes!

**Reply by DuyTC1811** [reply]
I was able to run with this command `docker-compose -f docker-materialize.yaml run cli `of yours
and i also realized where my mistake was ,`docker-materialize.yaml --> docker-materialize.yml`

thanks [bobbyiliev](https://github.com/bobbyiliev)

**Reply by bobbyiliev** [reply]
Hey @DuyTC1811, no problem at all! Happy to hear that you've got it all working!
